src/connection.toit

```diff
@@ -23,6 +23,9 @@ class Connection:
   current_writer_ := null
   current_reader_/reader.Reader? := null
 
+  // For testing.
```

Comment:
Is there no way to do this with sub-classing instead? It would be more general, I think.

Reply:
It's a bit hard to do with a subclass, because I would need
subclasses of Client, Server, and Connection, and would need
to make sure that the subclasses of Client and Server created
the right kind of Connection.  All in all probably more intrusive
than the current method.